modules/local/file_concatenator.nf

```diff
@@ -21,9 +21,9 @@ process FILE_CONCATENATOR {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     """
-    headeredFileConcatenator.pl $vcfs > snvs_${prefix}_raw.vcf
+    headeredFileConcatenator.pl $vcfs > snvs_${prefix}.vcf
```

Comment:
The raw VCF file is missing in the runs! Does this remove the file from the result structure?